src/main/java/meeseeks/parser/Parser.java

```diff
@@ -0,0 +1,70 @@
+package meeseeks.parser;
+
+import meeseeks.command.*;
+import meeseeks.task.Deadline;
+import meeseeks.task.Event;
+import meeseeks.task.Task;
+
+import java.time.LocalDateTime;
+
+public class Parser {
+    public static Command parse(String fullCommand) {
+        if (fullCommand.startsWith("todo")) {
+            return new AddCommand(fullCommand.substring(5), "todo");
+        } else if (fullCommand.startsWith("deadline")) {
+            String[] parts = fullCommand.split(" /by ", 2);
+            return new AddCommand(parts[0].substring(9), "deadline", parts[1]);
+        } else if (fullCommand.startsWith("event")) {
+            String[] parts = fullCommand.split(" /from | /to ");
+            return new AddCommand(parts[0].substring(6), "event", parts[1], parts[2]);
+        } else if (fullCommand.equals("list")) {
+            return new ListCommand();
+        } else if (fullCommand.startsWith("delete")) {
+            return new DeleteCommand(Integer.parseInt(fullCommand.substring(7)) - 1);
+        } else if (fullCommand.equals("bye")) {
+            return new ExitCommand();
+        }
+        return new UnknownCommand();
+    }
+
+    public static Task parseTaskFromFile(String line) {
```

Comment:
```suggestion
    public static Task parseTaskFromFileLine(String line) {
```
From the name, it is not clear if the function parses one task or many